python/cuml/dask/datasets/make_regression.py

```diff
@@ -0,0 +1,216 @@
+#
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import dask.array as da
+import cupy as cp
+
+
+def make_low_rank_matrix(n_samples=100, n_features=100, effective_rank=10,
+                         tail_strength=0.5, random_state=None, n_parts=1):
+    """ Generate a mostly low rank matrix with bell-shaped singular values
+
+    Parameters
+    ----------
+    n_samples : int, optional (default=100)
+        The number of samples.
+    n_features : int, optional (default=100)
+        The number of features.
```

Comment:
I would consider keeping the defsaults the same, like we did with make_blobs:
1) it doesn't assume which dimension users want to scale (eg. just because the user is using Dask, doesn't necessarily mean they are doing so to support larger number of features, and 
2) presenting the same defaults, with possible added options to suit distributed environments keeps the familiar face for the users. We've been doing this pretty consistently across all of our algorithms, both distributed and non-distributed.